src/ssl.c

```diff
@@ -31630,6 +31652,78 @@ int wolfSSL_CTX_use_PrivateKey(WOLFSSL_CTX *ctx, WOLFSSL_EVP_PKEY *pkey)
     defined(WOLFSSL_HAPROXY) || defined(OPENSSL_EXTRA) || \
     defined(HAVE_LIGHTY)) || defined(HAVE_EX_DATA) || \
     defined(WOLFSSL_WPAS_SMALL)
+CRYPTO_EX_cb_ctx* crypto_ex_cb_ctx_session = NULL;
+
+static int crypto_ex_cb_new(CRYPTO_EX_cb_ctx** dst, long ctx_l, void* ctx_ptr,
```

Comment:
Should these new `crypto_ex` functions have a leading `wolfssl_`?

Reply:
They are static and internal functions so I don't think that is necessary.